xlive/H2Config.cpp

```diff
@@ -686,6 +705,30 @@ static int interpretConfigSetting(char* fileLine, int version, int lineNumber) {
 				est_hide_ingame_chat = true;
 			}
 		}
+		else if (!H2IsDediServer && sscanf(fileLine, "controller_sensitivity =%d", &tempint1) == 1) {
+			if (est_sens_controller) {
+				duplicated = true;
+			}
+			else if (!(tempint1 >= 0)) {
```

Comment:
all these if/elseif/else seem like they can be one liners

also comments would be nice

Reply:
That's up to Glitchy to change

Reply:
Please explain how that can be simplified without losing current functionality.
Also those lines are self documenting. The only comments I'll ever add is a description for what that entire block of code does where the chain of if-else-ifs begin.

Reply:
you have a boolean called "duplicated" wtf does that even mean? Thats not self documenting, thats a poorly named bool

then TempInt1, another poorly named boolean.

This is not self documenting, this is only self documenting after you look around and figure out whatever everything means and where every value comes from.

Reply:
I don't care much about breaking the if/elseif/else into one liners, more so into starting to make the code more readable to anyone.